fixtures/minuit-une/ivl-photon.json

```diff
@@ -0,0 +1,798 @@
+        {
+          "dmxRange": [127, 127],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Strobe",
+          "comment": "Mid",
+          "menuClick": "center"
+        },
+        {
+          "dmxRange": [255, 255],
+          "type": "ShutterStrobe",
+          "shutterEffect": "Strobe",
+          "comment": "Long",
+          "menuClick": "center"
+        }
+      ]
+    },
+    "Gobo Beams Div": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 27],
+          "type": "WheelSlot",
+          "comment": "1 Beam"
+        },
+        {
+          "dmxRange": [28, 55],
+          "type": "WheelSlot",
+          "comment": "2 Beams"
+        },
+        {
+          "dmxRange": [56, 83],
+          "type": "WheelSlot",
+          "comment": "4 Beams"
+        },
+        {
+          "dmxRange": [84, 111],
+          "type": "WheelSlot",
+          "comment": "8 Beams"
+        },
+        {
+          "dmxRange": [112, 139],
+          "type": "WheelSlot",
+          "comment": "16 Beams"
+        },
+        {
+          "dmxRange": [140, 167],
+          "type": "WheelSlot",
+          "comment": "32 Beams"
+        },
+        {
+          "dmxRange": [168, 195],
+          "type": "WheelSlot",
+          "comment": "64 Beams"
+        },
+        {
+          "dmxRange": [196, 223],
+          "type": "WheelSlot",
+          "comment": "128 Beams"
+        },
+        {
+          "dmxRange": [224, 255],
+          "type": "WheelSlot",
+          "comment": "256 Beams"
+        }
+      ]
+    },
+    "Index": {
+      "fineChannelAliases": ["Index fine"],
+      "capability": {
+        "type": "Generic",
+        "comment": "index 360deg"
+      }
+    },
+    "Rotation": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 12],
+          "type": "Rotation",
+          "comment": "Index"
+        },
+        {
+          "dmxRange": [13, 122],
+          "type": "Rotation",
+          "comment": "Rotation fast CCW"
+        },
+        {
+          "dmxRange": [123, 133],
+          "type": "Rotation",
+          "comment": "Rotation stop"
+        },
+        {
+          "dmxRange": [134, 242],
+          "type": "Rotation",
+          "comment": "Rotation fast CW"
+        },
+        {
+          "dmxRange": [243, 255],
+          "type": "Rotation",
+          "comment": "Index"
+        }
+      ]
+    },
+    "Gobo Beam Size": {
+      "fineChannelAliases": ["Gobo Beam Size fine"],
+      "defaultValue": 65535,
+      "highlightValue": 65535,
+      "capability": {
+        "type": "WheelSlot",
+        "comment": "Beam 360deg"
+      }
+    },
+    "Reflector Zoom": {
+      "fineChannelAliases": ["Reflector Zoom fine"],
+      "capabilities": [
+        {
+          "dmxRange": [0, 0],
+          "type": "Zoom",
+          "comment": "Mirror"
+        },
+        {
+          "dmxRange": [255, 255],
+          "type": "Zoom",
+          "comment": "Reflector"
+        }
```

Comment:
I'm not sure what capability type this is, but I think it should be a range, not just discrete values at 0 and 255.